katas/content/distinguishing_states/ghz_w/index.md

```diff
@@ -0,0 +1,7 @@
+**Input:** $N \ge 2$ qubits (stored in an array of length $N$) which are guaranteed to be either in the [GHZ state](https://en.wikipedia.org/wiki/Greenberger%E2%80%93Horne%E2%80%93Zeilinger_state) or in the [W state](https://en.wikipedia.org/wiki/W_state).
```

Comment:
Same here, let's use definition and example instead of a link.
```suggestion
**Input:** $N \ge 2$ qubits (stored in an array of length $N$) which are guaranteed to be either in the GHZ state - an even superposition of $\ket{0...0}$ and $\ket{1...1}$ states - or in the W state we saw in the previous task. (For example, for $N = 3$ GHZ state is $\frac1{\sqrt2}(\ket{000} + \ket{111})$.
```